translations/README.md

```diff
@@ -2,7 +2,7 @@
 
 | File            | Language      |
 |-----------------|---------------|
-| Readme          | [:uk: English](../README.md) ￨ [:fr: Français](README.french.md) ￨ [:jp: 日本語](README.japanese.md) ￨ [🇬🇷 Ελληνικά](README.greek.md) ￨ [🇲🇽 Spanish](README.mx_spanish.md) ￨ [:brazil: Português](README.portuguese_br.md) ￨ [🇩🇪 Deutsch](README.german.md) ￨ [:ru: Русский](README.russian.md) ￨ [:indonesia: Indonesian](README.indonesian.md) ￨ [:bosnia_herzegovina: Bosnian](README.bosnian.md) ￨ [:cn: 中文](README.chinese.md)  |
+| Readme          | [:uk: English](../README.md) ￨ [:fr: Français](README.french.md) ￨ [:jp: 日本語](README.japanese.md) ￨ [🇬🇷 Ελληνικά](README.greek.md) ￨ [🇲🇽 Spanish](README.mx_spanish.md) ￨ [:brazil: Português](README.portuguese_br.md) ￨ [🇩🇪 Deutsch](README.german.md) ￨ [:ru: Русский](README.russian.md) ￨ [:indonesia: Indonesian](README.indonesian.md) ￨ [:bosnia_herzegovina: Bosnian](README.bosnian.md) ￨ [:cn: 中文](README.chinese.md)  | [:tr: Türkçe](README.turkish.md) ￨
```

Comment:
The Turkish one is not visible here:

![image](https://user-images.githubusercontent.com/5722022/75220319-28a07680-57c5-11ea-91fd-b48ae4bf2d5b.png)